docs/admin/multiple-schedulers.md

```diff
@@ -89,6 +89,31 @@ my-scheduler-lnf4s-4744f                       1/1       Running   0          2m
 You should see a "Running" my-scheduler pod, in addition to the default kube-scheduler
 pod in this list.
 
+If you want to run multiple-scheduler with leader election enabled, update the --leader-elect=true, --lock-object-namespace=`lock-object-namespace`,--lock-object-name=`lock-object-name` in the yaml file, and if the RBAC is enabled in your cluster, you need also update the `system:kube-scheduler` cluster role to add your scheduler name to the resourceNames of the rule that applied for the endpoints resource like this:
```

Comment:
This is a really big run-on sentence. I suggest splitting it up to make it easier for the reader to understand what they have to do:

"To run multiple-scheduler with leader election enabled, you must do the following:

First, update the following fields in your YAML file:

* --leader-elect=true
* --lock-object-namespace=`lock-object-namespace`
* --lock-object-name=`lock-object-name`

If RBAC is enabled on your cluster, you must update the `system:kube-scheduler` cluster role. Add you scheduler name to the resourceNames of the rule applied for endpoints resources, as in the following example:"